src/test/java/org/openhab/tools/analysis/checkstyle/test/ManifestExternalLibrariesCheckTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.tools.analysis.checkstyle.test;
+
+import java.io.File;
+
+import org.junit.Test;
+import org.openhab.tools.analysis.checkstyle.ManifestExternalLibrariesCheck;
+import org.openhab.tools.analysis.checkstyle.api.AbstractStaticCheckTest;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+import com.puppycrawl.tools.checkstyle.utils.CommonUtils;
+
+/**
+ * Test for the {@link ManifestExternalLibrariesCheck}
+ *
+ * @author Velin Yordanov - initial contribution
+ *
+ */
+public class ManifestExternalLibrariesCheckTest extends AbstractStaticCheckTest {
+    private static DefaultConfiguration configuration = createCheckConfig(ManifestExternalLibrariesCheck.class);
+    private static String TEST_FILE_NAME = "MANIFEST.MF";
+    private static final String MAIN_DIRECTORY = "manifestExternalLibrariesCheck";
+
+    @Override
+    protected DefaultConfiguration createCheckerConfig(Configuration config) {
+        DefaultConfiguration configParent = createCheckConfig(TreeWalker.class);
+        configParent.addChild(config);
+        return configParent;
+    }
+
+    @Test
+    public void shouldNotLogWhenBundleIsValid() throws Exception {
+        final String VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES = MAIN_DIRECTORY + File.separator + "validBundle"
+                + File.separator + "META-INF";
+        String[] warningMessages = CommonUtils.EMPTY_STRING_ARRAY;
+        String filePath = getPath(VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES + File.separator + TEST_FILE_NAME);
```

Comment:
The result of all suggestions would be lightweight test instead of all the repition:

```java
verifyNoErrorMessages(configuration, getManifestPath("validBundle"));
```